apps/modernization-ui/src/apps/page-builder/components/Navigation/Navigation.tsx

```diff
@@ -22,24 +23,27 @@ export const Navigation = ({ active }: Props) => {
                     onClick={() => goBack('pages')}>
                     <p>Page library</p>
                 </div>
-
-                <div
-                    className={`navigation__button ${active === 'condition-library' ? 'active' : ''}`}
-                    onClick={() => goBack('condition-library')}>
-                    <p>Condition library</p>
+                <div className={`navigation__button ${active === 'condition-library' ? 'active' : ''}`}>
+                    <a
+                        href={`${Config.nbsUrl}/ManageCondition.do?method=ViewConditionLib&actionMode=Manage&initLoad=true`}>
```

Comment:
suggestion:  I don't think the `Config.nbsUrl` is needed here, if url starts with `/nbs/` it should go to the correct place.

Reply:
fixed